app/templates/results.html

```diff
@@ -0,0 +1,93 @@
+<head>
+    <title> SilverScreen </title>
+
+    {# Load stylesheets #}
+    {% load static %}
+    <link rel="stylesheet" type="text/css" href="{% static "style.css" %}" />
+    <script type="text/javascript" src="{% static "js/jquery.js" %}"></script>
+    {# Load the tag library #}
+    {% load bootstrap3 %}
+
+    {# Load CSS and JavaScript #}
+    {% bootstrap_css %}
+    {% bootstrap_javascript %}
+</head>
+
+<body>
+	<nav class="navbar navbar-default">
+	  <div class="container-fluid">
+		<div class="navbar-header">
+		  <a class="navbar-brand" href="#">
+			SilverScreen
+		  </a>
+		       <form class="navbar-form navbar-left">
```

Comment:
This is the broken version of the navbar. This is fixed in my pull request. We should merge it ;)